hash/gbcolor.xml

```diff
@@ -1289,8 +1329,9 @@ Unreleased (music source code exists, possibly no prototypes exist)
 		<year>1999</year>
 		<publisher>Capcom</publisher>
 		<info name="serial" value="DMG-AUEJ-JPN"/>
+		<info name="gold" value="19991104"/>
 		<info name="release" value="19991224"/>
-		<info name="alt_title" value="爆走戦記 メタルウォーカーGB 鋼鉄の友情"/>
+		<info name="alt_title" value="爆走戦記 メタルウォーカーＧＢ ～鋼鉄の友情～"/>
```

Comment:
Do you have a reference for “GB” being styled in fullwidth?

Reply:
I'm using the official release spreadsheets.

Reply:
I've reverted the change.